spec/requests/webhooks/stripe/invoice_events_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'spec_helper'
+
+RSpec.describe 'Stripe invoice events', :vcr, type: :request do
+  before(:each) { bypass_event_signature payload }
+
+  describe 'invoice.payment_succeeded' do
+    let(:stripe_customer_id) {'cus_9aUcniAOYTXn42'} # matches invoice.payment_succeeded.json
+    let(:stripe_charge_id) {'ch_19HJd82VpjOYk6TmrzJdKLYR'} # matches invoice.payment_succeeded.json
+    let(:stripe_card_id) {'card_19HJd62VpjOYk6TmwKcemuLf'} # matches card related to invoice.payment_succeeded.json charge
+
+    let!(:organization) { create(:organization, stripe_customer_id: stripe_customer_id, org_type: 'M') }
+    let!(:market) { create(:market, stripe_customer_id: stripe_customer_id, organization_id: organization.id) }
+    let!(:market_2) { create(:market, stripe_customer_id: stripe_customer_id + 'KXM') }
+    let!(:credit_card) { create(:bank_account, bankable: market, stripe_id: stripe_card_id) }
```

Comment:
TODO: This still needs some clean up to use organizations instead of markets when storing the stripe_customer_id